packages/milliform/src/index.ts

```diff
@@ -16,10 +16,26 @@ interface Form {
   validate(): Errors
   reset(): void
 }
+// @TODO: finish Form return type
+
+// interface Form {
+//   values: {
+//     email: string
+//     password: string
+//   },
+//   setters: {
+//     setEmail(value: string) => void
+//     setPassword(value: string) => void
+//   }
+//   errors: { email?: string, password?: string } | null
+//   validate() => { email?: string, password?: string } | null
+//   reset() => void
+// }
+
 
 interface State {
   _meta: {
-    fields: Fields
+    fields: Fields<'email' | 'password'>
```

Comment:
Reminder that this should be replaced with a generic (I think), don't want to hardcoded the state shape to only the email and password fields.